etc/openqa/openqa.ini

```diff
@@ -116,7 +116,7 @@
 ## whether to clone CASEDIR or NEEDLES_DIR on the web UI if that var points to a Git repo
 #git_auto_clone = yes
 ## enable automatic updates of all test code and needles managed via Git
-#git_auto_update = yes
+#git_auto_update = no
```

Comment:
Would it be ok for you to keep git_auto_update enabled for now and move it to a separate follow-up PR? I think we can merge your PR more easily if we keep this behavior unchanged for now

Reply:
moved to a separate commit for now.